src/locales/cs/app.json

```diff
@@ -10,12 +10,19 @@
 	"language": {
 		"label": "Jazyky",
 		"en": "English",
-		"pl": "Polski",
-		"fr": "Français",
+		"cs": "Čeština",

```

Comment:
Could you please revert these changes (and similar ones in other language files)? 🙏 
The way i18next works is that it looks up a key, say `language.cs` in the current language file. If it cannot find a translated string there, it checks the "default" language - which is the `en` one. So it's enough to have these language names defined in one, central place - instead of duplicated all over. Which would make it more work to fix a typo, new translators think they have to update each language file, instead of only their own, etc.